### PR TITLE
New model support for pose estimation

### PR DESCRIPTION
**Pull request by ChloeWu1**
Add human-pose-estimation-0001 model support.
Works better under video input.

### CODE REVIEW

File: openvino_wrapper_lib/include/openvino_wrapper_lib/inferences/human_pose_estimation.hpp [hunk 1]
```diff
@@ -0,0 +1,172 @@
+// Copyright (c) 2018 Intel Corporation
```
Comment (LewisLiuPub) [inline openvino_wrapper_lib/include/openvino_wrapper_lib/inferences/human_pose_estimation.hpp:1]: 2018 -->2023

File: openvino_wrapper_lib/include/openvino_wrapper_lib/inferences/human_pose_estimation.hpp [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @brief A header file with declaration for HumanPoseEstimation Class
+ * @file vehicle_attribs_detection.hpp
```
Comment (LewisLiuPub) [inline openvino_wrapper_lib/include/openvino_wrapper_lib/inferences/human_pose_estimation.hpp:17]: set the correct file name.

File: openvino_wrapper_lib/include/openvino_wrapper_lib/inferences/human_pose_estimation.hpp [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+ */
+#ifndef OPENVINO_WRAPPER_LIB_INFERENCES_VEHICLE_ATTRIBS_DETECTION_HPP_
+#define OPENVINO_WRAPPER_LIB_INFERENCES_VEHICLE_ATTRIBS_DETECTION_HPP_
```
Comment (LewisLiuPub) [inline openvino_wrapper_lib/include/openvino_wrapper_lib/inferences/human_pose_estimation.hpp:20]: Set the macro align with file name.

File: openvino_wrapper_lib/include/openvino_wrapper_lib/inferences/human_pose_estimation.hpp [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+#include <memory>
+#include <vector>
+#include <string>
+#include "openvino_wrapper_lib/models/human_pose_estimation_model.hpp"
+#include "openvino_wrapper_lib/engines/engine.hpp"
+#include "openvino_wrapper_lib/inferences/base_inference.hpp"
+#include "openvino_wrapper_lib/inferences/openpose_decoder.hpp"
+#include "opencv2/opencv.hpp"
+// namespace
+
```
Comment (LewisLiuPub) [inline openvino_wrapper_lib/include/openvino_wrapper_lib/inferences/human_pose_estimation.hpp:31]: Don't keep many empty line in one place. One empty line is enough to separate different func blocks.

File: openvino_wrapper_lib/include/openvino_wrapper_lib/inferences/human_pose_estimation.hpp [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+
+
+namespace openvino_wrapper_lib
+{
+/**
+ * @class HumanPoseEstimationResult
```
Comment (LewisLiuPub) [inline openvino_wrapper_lib/include/openvino_wrapper_lib/inferences/human_pose_estimation.hpp:37]: make all comments correct and aligh with the code.

**Review by LewisLiuPub — CHANGES_REQUESTED** [review]
Hi, @ChloeWu1, Thanks for your quick response with the implementation. But could you update the comments correct before the review process is triggered?

File: openvino_wrapper_lib/include/openvino_wrapper_lib/inferences/human_pose_estimation.hpp (hunk 1 above)
Comment (ChloeWu1) [reply openvino_wrapper_lib/include/openvino_wrapper_lib/inferences/human_pose_estimation.hpp]: Fixed.

File: openvino_wrapper_lib/include/openvino_wrapper_lib/inferences/human_pose_estimation.hpp (hunk 2 above)
Comment (ChloeWu1) [reply openvino_wrapper_lib/include/openvino_wrapper_lib/inferences/human_pose_estimation.hpp]: Fixed.

File: openvino_wrapper_lib/include/openvino_wrapper_lib/inferences/human_pose_estimation.hpp (hunk 3 above)
Comment (ChloeWu1) [reply openvino_wrapper_lib/include/openvino_wrapper_lib/inferences/human_pose_estimation.hpp]: Fixed.

File: openvino_wrapper_lib/include/openvino_wrapper_lib/inferences/human_pose_estimation.hpp (hunk 4 above)
Comment (ChloeWu1) [reply openvino_wrapper_lib/include/openvino_wrapper_lib/inferences/human_pose_estimation.hpp]: Fixed.

File: openvino_wrapper_lib/include/openvino_wrapper_lib/inferences/human_pose_estimation.hpp (hunk 5 above)
Comment (ChloeWu1) [reply openvino_wrapper_lib/include/openvino_wrapper_lib/inferences/human_pose_estimation.hpp:34]: Fixed.

**Comment by LewisLiuPub** [comment]
sounds good. 
Could you update Readme and model.lst? @ChloeWu1

File: openvino_wrapper_lib/include/openvino_wrapper_lib/inferences/human_pose_estimation.hpp [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+        }
+    }
+
+private:
+    const std::vector<cv::Mat>& heatMaps;
+    float minPeaksDistance;
+    std::vector<std::vector<Peak>>& peaksFromHeatMap;
+    float confidenceThreshold = 0.5;
+};
+#endif  // OPENVINO_WRAPPER_LIB_INFERENCES_VEHICLE_ATTRIBS_DETECTION_HPP_
```
[the remark this answers is not in the document]
Comment (ChloeWu1) [reply openvino_wrapper_lib/include/openvino_wrapper_lib/inferences/human_pose_estimation.hpp]: Fixed.

**Comment by ChloeWu1** [comment]
Done. Update model.lst, Readme, quick start and docker instruction.

**Comment by huangjiafengx** [comment]
The path for test result:
\\shwde7333.ccr.corp.intel.com\Robotics\temp\robotguide-gm-planning video\test_4_code_merge\human_pose

**Pull request merged**